kibble/config/config_test.go

```diff
@@ -41,3 +39,21 @@ func TestLoadLanguagesConfigWithObjects(t *testing.T) {
 	assert.Equal(t, cfg.Languages["it"].Code, "it_IT")
 	assert.Equal(t, cfg.Languages["it"].Name, "Italian")
 }
+
+func TestLoadConfig(t *testing.T) {
+	sitePath = "./test/kibble-translations-key-true.json"
+	cfg := LoadConfig(false, "api-key", false)
+	assert.Equal(t, true, cfg.UseTranslationsApi)
+}
+
+func TestLoadConfigFalse(t *testing.T) {
+	sitePath = "./test/kibble-translations-key-false.json"
+	cfg := LoadConfig(false, "api-key", false)
+	assert.Equal(t, false, cfg.UseTranslationsApi)
+}
+
+func TestLoadConfigMissing(t *testing.T) {
+	sitePath = "./test/kibble-translations-key-missing.json"
+	cfg := LoadConfig(false, "api-key", false)
+	assert.NotContains(t, "useTranslationsApi", cfg.UseTranslationsApi)
```

Comment:
Is this test right?  `cfg.UseTranslationsApi` _does_ exist and it should have the boolean zero value (`false`).

Reply:
Yeah... https://pkg.go.dev/github.com/stretchr/testify/assert#NotContains is for strings, lists, and maps... not structs!

Reply:
Use this instead: https://pkg.go.dev/github.com/stretchr/testify/assert#Zero

Reply:
Actually, I want this to be `false` (sorry) so that later when we use it in a condition we only have to worry about `true`/`false`. Is that possible?

Perhaps you can [add a default value of `false` before the JSON unmarshal](https://github.com/shift72/kibble/blob/master/kibble/config/config.go#L47-L50)?

Reply:
The boolean zero value is false.

Reply:
As Jack mentioned [zero function](https://tour.golang.org/basics/12) takes false by default for boolean by default @stajs

Reply:
Okay then make it like this:

```
assert.Equal(t, false, cfg.UseTranslationsApi)
```